### ISSUE TITLE
Incorrectly getting "Click to show 2 definitions" message when Ctrl+hover over a variable in template 

### ISSUE THREAD

**Issue by goodmite** [angular/vscode-ng-language-service#299]
**tldr**: vscode sems to think that a variable is declared twice in TS, while its not.

**Details:**

This problem is occuring in both Angular v6 and v7 projects.

When I click + hover over a variable in template I get this:
<img width="418" alt="gh" src="https://user-images.githubusercontent.com/16303528/48686292-30946280-ebe1-11e8-873c-3cdd9a5712f2.png">

When I Cltr + click over a variable in template I get this:
![image](https://user-images.githubusercontent.com/16303528/48686306-4b66d700-ebe1-11e8-8405-8194af79c1f5.png)

app.component.html:
`<input type="text" [value]="testVar">`

app.component.ts
```
import {Component, OnInit} from '@angular/core';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.scss']
})
export class AppComponent {
  testVar = 'hello';
}
```

**My package.json**
```
{
  "name": "angular-material7-test1",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~7.0.0",
    "@angular/cdk": "~7.0.4",
    "@angular/common": "~7.0.0",
    "@angular/compiler": "~7.0.0",
    "@angular/core": "~7.0.0",
    "@angular/forms": "~7.0.0",
    "@angular/http": "~7.0.0",
    "@angular/material": "^7.0.4",
    "@angular/platform-browser": "~7.0.0",
    "@angular/platform-browser-dynamic": "~7.0.0",
    "@angular/router": "~7.0.0",
    "au-input": "^3.0.1",
    "core-js": "^2.5.4",
    "hammerjs": "^2.0.8",
    "rxjs": "~6.3.3",
    "saturn-datepicker": "^6.1.1",
    "zone.js": "~0.8.26"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.10.0",
    "@angular/cli": "~7.0.5",
    "@angular/compiler-cli": "~7.0.0",
    "@angular/language-service": "~7.0.0",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "~4.5.0",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~3.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "typescript": "~3.1.6"
  }
}

```

OS: Windows 10

**Comment by SteveVanOpstal** [comment]
I figured out that some plugins may interfere with the language service. The `vismalietuva.vscode-angular-support` plugin caused this issue to occur for me.

**Comment by kyliau** [comment]
@goodmite Could you please make sure no other Angular extensions are running and try this again?

**Comment by kyliau** [comment]
Closing this, I believe this is not an issue with the Angular extension itself. Please reopen otherwise.
